pyOCD/board/mbed_board.py

```diff
@@ -87,6 +87,7 @@ def __init__(self, name, target, binary):
                 "9012": BoardInfo(  "Seeed Tiny BLE",       "nrf51",            "l1_nrf51.bin",         ),
                 "9900": BoardInfo(  "Microbit",             "nrf51",            "l1_microbit.bin",      ),
                 "C004": BoardInfo(  "tinyK20",              "k20d50m",          "l1_k20d50m.bin",       ),
+                "7402": BoardInfo(  "FRDM-K64F",            "k64f",             "l1_k64f.bin",          ),
```

Comment:
Please order this by board ID. Also, you'll want to change the board name to reflect that it is a "mbed 6LoWPAN Border Router HAT".